### DISCUSSION TITLE
Zen Sidebar Web Panel Features

### DISCUSSION THREAD

**Discussion by Muko-Tabi** [zen-browser/desktop#902] [Ideas]
# Enhancement: Improved Sidebar Web Panel Functionality for Zen Browser
- Some proposals I gather around the place into one that I think would be great for Zen Browser.

### 1. Container Support
- Allow web panels to run in separate containers for improved organization.

### 2. Adjustable Width
- Implement the ability to resize web panels by dragging their edges.

### 3. Customizable Grouping
- Enable users to organize web panels into folders.
- Allow customization of folder names, icons, and colors.

### 4. Flexible Positioning
- Provide an option to switch web panels between the left and right sides of the browser window.

### 5. Collapsible View Options
- Add a toggle to switch between:
  a) An always-visible mode with small icons representing each panel (Like in Floorp).
  b) A hidden mode that can be expanded when needed (Like the current default).

## Expected Benefits
These enhancements would significantly improve the utility and user-friendliness of the Sidebar Web Panel feature in Zen, offering users more control over their browsing experience compare to other browsers (and for the comparing table too).

**Comment by thibaultmol** [comment]
I (but maybe I'm alone in this) the ability to temp pop it into a large window or even fullscreen. (certain things you might want to look into it at a large window)
